### PR TITLE
Issue 437

### PR DESCRIPTION
**Pull request by larioj**

### CODE REVIEW

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/circe/Encoders.scala [hunk 1]
```diff
@@ -115,6 +115,11 @@ object Encoders {
       s"Package file [$fileName] not found"
     case PackageFileNotJson(fileName, parseError) =>
       s"Package file [$fileName] is not JSON: $parseError"
+    case UnableToParseMarathonAsJson(parseError) =>
+      s"Unable to parse filled-in Marathon template as JSON; there " +
+        s"may be an error in the package's Marathon template or default " +
+        s"configuration options, or in the installation request's options. " +
+        s"Parsing error was: $parseError"
```
Comment (cruhland) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/circe/Encoders.scala]: The `s` prefix on the string literals is for [string interpolation](http://docs.scala-lang.org/overviews/core/string-interpolation.html) and is only needed for the last line.

**Comment by cruhland** [comment]
@larioj LGTM, with one minor comment.

**Comment by cruhland** [comment]
@larioj could you clean up the commits on this PR too so I can merge it easily?

**Comment by larioj** [comment]
Will do

**Comment by BenWhitehead** [comment]
Fixes #437

**Pull request merged**